assets/database/db.json

```diff
@@ -18922,8 +18922,8 @@
 {"id":187815,"name":"Enchant Cloak - Greater Fire Resistance","icon":"spell_fire_sealoffire"}
 ],
 "encounters":[
-{"path":"Default/Raid Target","targets":[{"path":"Default/Raid Target","target":{"id":31146,"name":"Raid Target","level":88,"mobType":7,"stats":[0,0,0,0,0,0,0,0,0,0,0,0,650,0,0,0,0,0,0,0,0,0,11977,0,120016403,0,0],"minBaseDamage":210000,"damageSpread":0.4,"swingSpeed":2.5}}]},
-{"path":"Default/Movement","targets":[{"path":"Default/Movement","target":{"id":31147,"name":"Movement","level":88,"mobType":7,"stats":[0,0,0,0,0,0,0,0,0,0,0,0,650,0,0,0,0,0,0,0,0,0,11977,0,120016403,0,0],"minBaseDamage":210000,"damageSpread":0.4,"swingSpeed":2.5,"targetInputs":[{"inputType":1,"label":"Movement Interval","tooltip":"How often the player will move in seconds","numberValue":10},{"inputType":1,"label":"Reaction Time","tooltip":"How long the player can wait for casts to finish before moving in seconds","numberValue":1.5},{"inputType":1,"label":"Yards","tooltip":"How many yards the player moves","numberValue":5}]}}]},
+{"path":"Default/Raid Target","targets":[{"path":"Default/Raid Target","target":{"id":31146,"name":"Raid Target","level":93,"mobType":7,"stats":[0,0,0,0,0,0,0,0,0,0,0,0,650,0,0,0,0,0,0,0,0,0,24835,0,120016403,0,0],"minBaseDamage":210000,"damageSpread":0.4,"swingSpeed":2.5}}]},
+{"path":"Default/Movement","targets":[{"path":"Default/Movement","target":{"id":31147,"name":"Movement","level":93,"mobType":7,"stats":[0,0,0,0,0,0,0,0,0,0,0,0,650,0,0,0,0,0,0,0,0,0,24835,0,120016403,0,0],"minBaseDamage":210000,"damageSpread":0.4,"swingSpeed":2.5,"targetInputs":[{"inputType":1,"label":"Movement Interval","tooltip":"How often the player will move in seconds","numberValue":10},{"inputType":1,"label":"Reaction Time","tooltip":"How long the player can wait for casts to finish before moving in seconds","numberValue":1.5},{"inputType":1,"label":"Yards","tooltip":"How many yards the player moves","numberValue":5}]}}]},
```

Comment:
The `minBaseDamage` should be way higher as well for a representative MoP boss target.